src/sql/store.py

```diff
@@ -175,5 +178,48 @@ def _flatten(elements):
     return [element for sub in elements for element in sub]
 
 
+def store_snippet_as_sql(sql_command, snippet_name):
+    """
+    Store snippet as a SQl file
+
+    Parameters
+    ----------
+    command : str
+        query to be saved as the snippet .
+
+    saved_snippet : str
+        Name of the saved snippet
+    """
+
+    snippets_dir = "jupysql-snippets"
+    snippet_path = Path(snippets_dir) / f"{snippet_name}.sql"
+    snippet_path.parent.mkdir(parents=True, exist_ok=True)
+    with open(snippet_path, "w") as file:
+        file.write(sql_command)
+    message = (
+        """<div style="border: 2px solid #e8e8e8; padding: 10px; margin: 5px 0; """
+        """background-color: #f7f7f7; font-family: Arial, sans-serif;">"""
+        """<strong>Note:</strong> Manual editing of .sql files may not be """
+        """reflected when reopening the notebook. <br>Please edit snippets directly """
+        """ in the notebook to ensure consistency."""
+        """</div>"""
+    )
+
+    display(HTML(message))
+
+
+def load_snippet_from_sql(store):
+    snippets_dir = "jupysql-snippets/"
+    if os.path.exists(snippets_dir) and os.path.isdir(snippets_dir):
+        snippet_files = glob.glob(snippets_dir + "*.sql")
+        snippet_names = [filename[len(snippets_dir) : -4] for filename in snippet_files]
```

Comment:
It's a bit unclear to me what's the logic here. Can you please add a comment and docsrting?

Reply:
Added  a. docstring here